src/Slider.js

```diff
@@ -24,11 +20,6 @@ class Slider extends React.Component {
             <input
                 className={classes}
                 type="range"
-                min={min}
-                max={max}
-                value={value}
-                tabIndex="0"
```

Comment:
I just tried it : without `tabIndex=0`, tabbing through still works properly.

tabindex=0 should be the default for any focusable element